src/atomate2/openmm/flows/core.py

```diff
@@ -0,0 +1,100 @@
+from src.atomate2.openmm.jobs.core import (
+    OpenMMSetFromInputMoleculeSpec,
+    OpenMMSetFromDirectory,
+    EnergyMinimizationMaker,
+    NPTMaker,
+    AnnealMaker,
+    NVTMaker,
+)
+from typing import List, Union, Optional, Dict
+from pathlib import Path
+
+from dataclasses import dataclass
+
+from pydantic import Field
+from jobflow import Maker, Flow, job
+
+from pymatgen.io.openmm.schema import InputMoleculeSpec
+
+
+@dataclass
+class ProductionMaker(Maker):
+    """
+
+
+    """
+    name: str = "production"
+
+    input_maker: OpenMMSetFromDirectory = Field(default_factory=OpenMMSetFromDirectory)
+    energy_maker: EnergyMinimizationMaker = Field(default_factory=EnergyMinimizationMaker)
+    npt_maker: NPTMaker = Field(default_factory=NPTMaker)
+    anneal_maker: AnnealMaker = Field(default_factory=AnnealMaker)
+    nvt_maker: NVTMaker = Field(default_factory=NVTMaker)
+
+    def make(self, input_dir: Union[str, Path]):
+
+        openmm_set_from_dir_job_0 = self.input_maker.make(input_dir=input_dir)
+
+        energy_job = self.energy_maker.make(input_set=openmm_set_from_dir_job_0.output, output_dir=input_dir)
+
+        openmm_set_from_dir_job_1 = self.input_maker.make(input_dir=input_dir)
+
+        pressure_job = self.npt_maker.make(input_set=openmm_set_from_dir_job_1.output, output_dir=input_dir)
+
+        openmm_set_from_dir_job_2 = self.input_maker.make(input_dir=input_dir)
+
+        anneal_job = self.anneal_maker.make(input_set=openmm_set_from_dir_job_2.output, output_dir=input_dir)
+
+        openmm_set_from_dir_job_3 = self.input_maker.make(input_dir=input_dir)
+
+        production_job = self.nvt_maker.make(input_set=openmm_set_from_dir_job_3.output, output_dir=input_dir)
```

Comment:
I think you are currently overwriting the original input set whenever you run this, making future runs fail earlier. At least, that's what's happening for me.

Even if we make the `InputMaker` accept an `input_dir` is it still possible to have everything else input and output `input_sets`? Do we need to call `input_maker` every time?

I'm also curious about the `ouput_dir`, we will need a way to log output eventually but we could try to do it entirely in-memory and then eventually write it to a mongo-like database, rather than write it on the filesystem.